src/metal/global.js

```diff
@@ -0,0 +1,11 @@
+/* global global */
+
+let globalNamespace;
+
+if (typeof global === 'undefined') {
+  globalNamespace = window;
+} else {
+  globalNamespace = global;
+}
+
+export default globalNamespace;
```

Comment:
Nah. If global is undefined that will throw. Also, no logic in an export statement.